erpnext/docs/user/manual/en/selling/articles/applying-discount.md

```diff
@@ -1,31 +1,25 @@
-<h1>Applying Discount</h1>
+#Applying Discount
 
-There are two ways Discount can be applied on an items in the sales transactions.
+There are several ways Discount can be applied on an items in the sales transactions.
 
 #### 1. Discount on "Price List Rate" of an item
 
-In the Item table of transaction, after Price List Rate field, you will find Discount (%) field. Discount Rate applied in this field will be applicable on the Price List Rate of an item.
+In the Item table, after Price List Rate field, you will find Discount (%) field. Discount Rate applied will be applicable on the Price List Rate of an item.
```

Comment:
Simplified:
*You can find the Discount (%) field in the Item table. Discount is applied on the Price List Rate to get the selling Rate of the Item.*